src/split.jl

```diff
@@ -31,21 +31,38 @@ findwhen(ta::TimeArray{Bool, 1}) = ta.timestamp[find(ta.values)]
 
 ###### head, tail ###########
 
-function head(ta::TimeArray, n::Int=1)
+function head(ta::TimeArray, n::Int=6)
     ncol          = length(ta.colnames)
     new_timestamp = ta.timestamp[1:n]
     new_values    = ta.values[1:n, 1:ncol]
     TimeArray(new_timestamp, new_values, ta.colnames, ta.meta)
 end
 
-function tail(ta::TimeArray, n::Int=1)
+function tail(ta::TimeArray, n::Int=6)
     ncol          = length(ta.colnames)
     tail_start = length(ta)-n+1
     new_timestamp = ta.timestamp[tail_start:end]
     new_values    = ta.values[tail_start:end, 1:ncol]
     TimeArray(new_timestamp, new_values, ta.colnames, ta.meta)
 end
 
+###### first, last ###########
+
+function first(ta::TimeArray)
```

Comment:
Maybe we should overload `Base.first` and `Base.last`?

Reply:
and I think this function should reuse `head`
```julia
Base.first(ta::TimeArray)  = head(ta, 1)
Base.last(ta::TimeArray)  = tail(ta, 1)
```

Reply:
Didn't think about it, good idea.